tests/test_inbox.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+from flask import url_for
+from flask.testing import FlaskClient
+
+from hushline.db import db
+from hushline.model import Message, User, Username
+
+
+@pytest.mark.usefixtures("_authenticated_user")
+def test_delete_message(client: FlaskClient, user: User) -> None:
```

Comment:
if we're adding tests for this it would be good to make sure that User A can't delete User B's messages, but this isn't a blocker to merging.

Reply:
done